hub/src/Website/SolarSENSE/models.py

```diff
@@ -27,3 +27,11 @@ def __str__(self):
     name = models.CharField(max_length=255)
     content = models.FileField()
     tags = TaggableManager(blank=True)
+
+
+class Collections(models.Model):
```

Comment:
You might want to consider renaming the Collections object to something less generic relating to the fields in holds like SensorCollection or something.